CONTRIBUTING.md

```diff
@@ -1,38 +1,115 @@
+python hazenlib/__init__.py snr tests/data/snr/GE
+```
+
+## 2) Developer Process for Contributing
+
+Follow these steps to make a contribution to hazen:
+
+1. Check the current [Issues](https://github.com/GSTT-CSC/hazen/issues) to see if an Issue already exists for your 
+contribution.
+2. If there is no existing Issue, create a new Issue for your contribution:
+   - Select the `Bug report` or `Feature request` template
+   - Fill in the Issue according to the template
+   - Add relevant Labels to the issue: `Enhancement`, `Bug`, `MRI`, `DICOM`, etc
+3. Create a new branch from `main`
+   - Name the branch with the issue number and a short description, e.g.: `123-snr-bugfix`
+4. Make your code changes (see guidance above)
+5. Perform unit tests on your machine: `pytest tests/`
+6. Create a [Pull Request](https://github.com/GSTT-CSC/hazen/pulls)
+   - Describe your changes
+   - Describe why you coded your change this way
+   - Describe any shortcomings or work still to do
+   - Cross-reference any relevant Issues
+7. One of the hazen team members will review your Pull Request and then:
+   - Ask questions
+   - Request any changes
+   - Merge into `main` – thank you and congratulations on contributing to hazen!
+
+
+## 3) Release Process
+
+The Release Process involves approving and then merging all Pull Requests identified for the new release of hazen. 
+Follow these steps for a new Release:
+
+For each PR:
+1. Ensure you understand the changes being introduced by the PR
+   - Request further information or changes from the Developer where necessary 
+2. When the PR is ready to merge, ensure all automated GitHub tests and checks are passing
+   - Investigate any which are failing
+3. Merge the PR into the `main` branch
+   - Provide a brief description of changes in the PR textbox as this forms the Release Notes 
+
+For a new release:
+4. Close all related Issues resolved by the merged PRs
+5. **Important**: Update version numbers across the repo:
+   - Update version number in `hazenlib/_version.py`
+     - (This is automatically propagated into `docs/source/conf.py`, `hazenlib/__init__.py` and `setup.cfg`)
+   - Update version number in `CITATION.cff`
+6. Create a [new Release](https://github.com/GSTT-CSC/hazen/releases)
+   - Create a tag equal to the version number, e.g. 1.2.1
+   - Select `main` as the Target branch from which to create the Release
+   - Select "Generate release notes" to automatically pull in the Merge Commit messages
+   - Amend the Release Notes:
+     - Add any other changes not automatically generated
+     - Edit Release Notes for typos and formatting consistency
+7. Publish Release
+   - This will trigger the [publishing GitHub Actions](https://github.com/GSTT-CSC/hazen/tree/main/.github/workflows) 
+   which push the released version to DockerHub and [PyPI](https://pypi.org/project/hazen/).
+8. Send a Release email to the hazen users!
```

Comment:
Maybe we should include a template or screenshot of previous Release emails?

Reply:
Good idea. Will add.